zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/JsonSerializers.java

```diff
@@ -194,6 +194,19 @@ static Annotation parseAnnotation(JsonParser parser) throws IOException {
     return Annotation.create(timestamp, value);
   }
 
+  private static void skipUnknownValue(JsonParser parser) throws IOException {
```

Comment:
Can we rewrite this for less line of code,

if(value == JsonToken.START_OBJECT || value == JsonToken.START_ARRAY){
    while(parser.nextValue() != (value == Json.START_OBJECT ? JsonToken.END_OBJECT : JsonToken.END_ARRAY)){
   skipUnknownValue(parser);
}
}

Let me know, if I do something wrong in this.

Reply:
@ankit-gautam23 as this PR is not updated recently, are you up to raising a new one. including a test to prove that the skipping works?

Reply:
@codefromthecrypt  Sure will include that too, Thanks for considering and Sorry for late response